src/test/test-random-util.c

```diff
@@ -51,6 +55,50 @@ static void test_rdrand(void) {
         }
 }
 
+#define TOTAL 100000
+
+static void test_random_u64_range_one(unsigned mod) {
```

Comment:
maybe may "mod" size_t too? you scale a VLA with it, and you iterate through it with `size_t i`, hence maybe just make this size_t too?

Reply:
Is it OK if I leave it as is? After all it's test code with a hardcoded value. Even if somebody wanted to boil some sea and changed this to a really large number, the compiler will refuse as soon as the number overflows `unsinged`.